torchdynamo/variables/nn_module.py

```diff
@@ -105,13 +105,20 @@ def var_getattr(self, tx, name):
         if not self.source:
             unimplemented("GETATTR with no source")
 
-        if name in base_dict:
+        # class attribute has the highest priority
+        try:
+            subobj = inspect.getattr_static(base.__class__, name)
+            class_member = False
```

Comment:
I still use ```class_member``` but I think it's a confusing name, ```object_member``` should be better? Correct me if I'm wrong. @jansel